### DISCUSSION TITLE
cambiar la ruta del hls

### DISCUSSION THREAD

**Discussion by Igmam21** [owncast/owncast#2787] [General Owncast discussion]
saludos amigos. cuando inicio el **owncast**, este me crea una carpeta llama "hls"   y la configuracion de coneccion es:  **rtmp://ip-o-nombe-se-server:1935/live**    y el archivo **m3u8**  con el nombre de **stream.m3u8**

mi pregunta es.... como puedo cambiarle el nombre al archivo **stream.m3u8**   por uno personalizado y tambien el noombre de la carpeta hls por otro nombre...   

greetings friends. When I start **owncast**, it creates a folder called "hls" and the connection configuration is: **rtmp://ip-or-name-of-server:1935/live** and the file * *m3u8** with the name of **stream.m3u8**

My question is... how can I change the name of the file **stream.m3u8** to a custom one and also the name of the hls folder to another name...

**Comment by gabek** [comment]
Hi there. I'm afraid there's no supported way to change that as there's very few use cases where people would need something like that for normal operation. You'd need to edit both the frontend and backend source code of Owncast to make specific changes such as that.

**Reply by Igmam21** [reply]
is that I already tried to change it in the source code files. reboot, but no changes are applied

**Reply by gabek** [reply]
You would need to edit the backend source and recompile it with your changes. Changing the frontend simply wouldn't be enough.
